github_helper.py

```diff
@@ -0,0 +1,92 @@
+"""github_helper provides several methods to help interface with Github
+
+github_helper has one class, GithubHelper, with the below methods.
```

Comment:
This documentation should go on the class/methods being documented using the standard PyDoc formats.